### ISSUE TITLE
should rangeChanged trigger onTimeSelected?

### ISSUE THREAD

**Issue by elewin** [twinssbc/Ionic-Calendar#132]
Right now, onTimeSelected will trigger when you click a date box on the calendar, but swiping the calendar to a new month will also change the selected date but it will not invoke onTimeSelected since its (I assume) only looking for click events. However, if there is some functionality that depends on onTimeSelected to keep some other codepoint synced up with the date that is currently selected on the calendar, it will lag behind on a month-swipe since it wont have any awareness of the auto-selected date.

Is there a quick and easy way to tie onTimeSelected  (or some other event) to fire any time a date is selected, not just by being clicked but also if a new date is selected automatically from changing months?

**Comment by twinssbc** [comment]
@elewin Yes, this feature was raised in the Ionic2-Calendar and get added. But I didn't add it in Ionic-Calendar, as no one raised it before.
I will add it. But just to clarify the onTimeSelected event will be triggered after loading the events, because onTimeSelected provides the events happen on that date. When rangeChanged was triggered, the events hasn't been fetched.
